core-plugins/widgets/HDFSAction-action.json

```diff
@@ -0,0 +1,40 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "Properties",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Source Path",
+          "name": "sourcePath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Destination Path",
+          "name": "destPath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Wildcard",
+          "name": "fileRegex"
+        },
+        {
+          "widget-type": "select",
+          "label": "Stop Pipeline Upon Error",
```

Comment:
you've labeled it as stop on error, but a true value continues on error, it doesn't stop.